Library/Formula/wireshark.rb

```diff
@@ -3,13 +3,9 @@ class Wireshark < Formula
   homepage "https://www.wireshark.org"
 
   stable do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
-    sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
-
-    # Removes SDK checks that prevent the build from working on CLT-only systems
-    # Reported upstream: https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=9290
-    patch :DATA
+    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2"
```

Comment:
> I could build 2.0.0 successfully on fresh Mavericks after xcode-select --install.
There is something different from CI environment probably. How can I build the same env as Jenkins?

Nothing at all there. Mavericks CI has both Xcode and CLT installed, up-to-date, etc etc.